src/pages/slots/CreateSlot.tsx

```diff
@@ -0,0 +1,63 @@
+import { MainContainer, Label, Input, Button, Select, InputContainer, EditContainer, EditCapacity, TitleContainer, Title, ScreenContainer } from "./CreateSlot.styles";
+import EditIcon from "../../assets/edit-icon.png";
+import EditSlotForm from "./forms/EditCapacityForm";
+import { useRef, useState } from "react";
+import type { ReactNode } from "react";
+import AddIcon from "../../assets/add-icon.svg";
+import Modal from "../../components/modal/Modal";
+import BackIcon from "../../assets/back-icon.png";
+import { useNavigate } from "react-router-dom";
+
+function CreateSlot() {
+
+
+
+    const DaysOfWeek = ["Lunes", "Martes", "Miércoles", "Jueves", "Viernes", "Sábado", "Domingo"];
+
+    const [modal, setModal] = useState<ReactNode | null>(null);
+
+    const editCapacityRef = useRef<any>(null);
+
+    const navigate = useNavigate();
+
+    return (
+        <MainContainer>
+            <TitleContainer>
+                <Title>
+                    <img
+                        src={BackIcon}
+                        alt="back-icon"
+                        onClick={() => navigate(-1)}
+                        style={{ cursor: "pointer" }}
+                    ></img>
```

Comment:
Aca lo mismo con respecto al cierre

```suggestion
                    />
```